tests/api.c

```diff
@@ -16781,6 +16846,26 @@ static void test_wc_PKCS7_EncodeSignedData(void)
     pkcs7->hashOID = 0; /* bad hashOID */
     AssertIntEQ(wc_PKCS7_EncodeSignedData(pkcs7, output, outputSz), BAD_FUNC_ARG);
 
+#if defined(HAVE_PKCS7) && defined(HAVE_PKCS7_RSA_RAW_SIGN_CALLBACK) && \
+    !defined(NO_RSA) && !defined(NO_SHA256)
+    /* test RSA sign raw digest callback, if using RSA and compiled in.
+     * Example callback assumes SHA-256, so only run test if compiled in. */
+    AssertNotNull(pkcs7 = wc_PKCS7_New(HEAP_HINT, devId));
```

Comment:
leaves the previous pkcs7 pointer stranded without free'ing first

Reply:
Thanks, fixed!